### ISSUE TITLE
Create Interval operation makes huge interval

### ISSUE THREAD

**Issue by idenisovs** [idenisovs/money-saver#9]
Preconditions:

1. There is old interval, which end in last year;
2. Between new and old intervals is more than week.

Steps to reproduce:

1. Observe, that there is old, ended interval for last year;
2. Create new interval in new year;
3. Move back to last interval;
4. Observe, that it changed it`s bound to huge period of time;

**Comment by idenisovs** [comment]
Workflow of _bl/intervals/create-interval.js_ should be reworked!

This bug occurs on  **create-interval.js:64**

```JavaScript
latestInterval.end = moment(interval.start).subtract(1, 'days').valueOf();
```

**Comment by idenisovs** [comment]
Should be fixed by this commit: [7f0e603](https://github.com/idenisovs/money-saver/commit/7f0e6039d3d2c97254b11a15e21cfe4770c6db97)
